tests/integration/cattletest/core/test_allocation.py

```diff
@@ -67,20 +68,21 @@ def test_spread(admin_client, sim_context, sim_context2, sim_context3):
 
     counts = []
     for i, h in enumerate(hosts):
-        h = admin_client.update(h, {
+        h = internal_test_client.update(h, {
             'computeFree': 10000000
         })
         counts.append(h.computeFree)
 
     containers = []
     for _ in range(len(hosts) * count):
-        c = admin_client.create_container(imageUuid=sim_context['imageUuid'])
+        c = admin_client.create_container(
```

Comment:
This makes sense. i'll refactor it further to just use only internal_test_client.